pkg-manager/core/src/install/index.ts

```diff
@@ -1037,6 +1036,7 @@ const _installInContext: InstallFunction = async (projects, ctx, opts) => {
       workspacePackages: opts.workspacePackages,
       patchedDependencies: opts.patchedDependencies,
       lockfileIncludeTarballUrl: opts.lockfileIncludeTarballUrl,
+      useExperimentalNpmjsFilesIndex: opts.useExperimentalNpmjsFilesIndex,
```

Comment:
make it true only if `opts.lockfileOnly` is true too
```suggestion
      useExperimentalNpmjsFilesIndex: opts.lockfileOnly && opts.useExperimentalNpmjsFilesIndex,
```

Reply:
I don't see it resolved

Reply:
it was resolved, i think I lost it when I reverted one of the other changes. Recommited